crates/sui-types/src/object.rs

```diff
@@ -615,7 +616,8 @@ impl Display for Owner {
 }
 
 #[derive(Eq, PartialEq, Debug, Clone, Deserialize, Serialize, Hash)]
-pub struct Object {
+#[serde(rename = "Object")]
```

Comment:
What does this rename do?

Reply:
Keeps object digests stable - the digest implementation includes the name of the struct as determined by serde.